.github/workflows/test.yml

```diff
@@ -14,6 +14,7 @@ on:
   pull_request:
     branches:
     - main
+    - dev/4.x
```

Comment:
do you think it's necessary to add `3.x` here to run tests when doing maintenance PRs for bug fixing and such?

Reply:
Yes, great point. I'll add that. And I guess we should remove dev/4.x before merging.

Reply:
Added in commit 5ae45889.
(I'm not actually sure if instead we'll need this change to test.yml on the *3.x* branch for it to be used for pushes and PR targetting 3.x. We shall see.)